flows/utils.py

```diff
@@ -64,3 +66,41 @@ def remove_translated_suffix(file_name: str) -> str:
     E.g. "CCLW.executive.1.1_en_translated" -> "CCLW.executive.1.1"
     """
     return re.sub(r"(_translated(?:_[a-zA-Z]+)?)$", "", file_name)
+
+
+# TODO: Add test
```

Comment:
Holding off on writing tests until some of this work gets reviewed in case we go down a different path.